.github/workflows/website-deploy.yml

```
@@ -0,0 +1,20 @@
+name: Deploy Website
+on:
```

yamllint (1.29.0-1)

[warning] truthy value should be one of [false, true]

(truthy)